charger/eebus.go

```diff
@@ -143,34 +160,38 @@ func (c *EEBus) showCurrentChargingSetup() {
 	if prevComStandard != data.EVData.CommunicationStandard {
 		c.communicationStandard = data.EVData.CommunicationStandard
 		timestamp := time.Now()
-		c.log.WARN.Println("!! ", timestamp.Format("2006-01-02 15:04:05"), " ev-charger-communication changed from ", prevComStandard, " to ", data.EVData.CommunicationStandard)
+		c.log.TRACE.Println("!! ", timestamp.Format("2006-01-02 15:04:05"), " ev-charger-communication changed from ", prevComStandard, " to ", data.EVData.CommunicationStandard)
```

Comment:
brauchen wir die Timestamps jeweils doppelt zum Timestamp im Log?

Reply:
Beim Warning war es sinnvoll da es in der UI angezeigt wurde, jetzt nicht mehr. Wird entfernt